app/src/main/java/nodomain/freeyourgadget/gadgetbridge/activities/DebugActivity.java

```diff
@@ -264,6 +271,18 @@ private void testNotification() {
         nManager.notify((int) System.currentTimeMillis(), ncomp.build());
     }
 
+    // Return user input from screen
+    private String GetUserMessage()
```

Comment:
Nit: can you name it with a lower case first character, like getUserMessage()? Thanks :-)